plugin.video.tweakers/addon.xml

```diff
@@ -2,12 +2,14 @@
 <addon 
 	id="plugin.video.tweakers" 
 	name="Tweakers" 
-	version="1.1.8"
-	provider-name="Skipmode A1,Amelandbor,ninovanhooff,Martijn">
+	version="1.1.9"
+	provider-name="Skipmode A1">
   <requires>
-    <import addon="xbmc.python"                 version="2.14.0"/>
-	<import addon="script.module.beautifulsoup" version="3.0.8"/>
-    <import addon="script.module.requests"      version="2.4.3"/>
+    <import addon="xbmc.python"                     version="2.14.0"/>
+	<import addon="script.module.beautifulsoup4"    version="4.5.3"/>
```

Comment:
@skipmodea1 ERROR: Version mismatch for addon script.module.beautifulsoup4. Required: 4.5.3, Available: 4.3.2